CHANGELOG.md

```diff
@@ -72,6 +72,7 @@
 - Fixed CVE-2020-14040 by updating to `golang.org/x/text/0.3.3`. ([#3458](https://github.com/operator-framework/operator-sdk/pull/3458))
 - The `scorecard` subcommand now removes existing pods if the `--wait-time` deadline is exceeded and `--skip-cleanup=false` (the default). Fixes [#3419](https://github.com/operator-framework/operator-sdk/issues/3419). ([#3526](https://github.com/operator-framework/operator-sdk/pull/3526))
 - Fixed a bug in scorecard that caused tests to fail with permission errors when loading the bundle. ([#3428](https://github.com/operator-framework/operator-sdk/pull/3428))
+- Fixed debug logging in the `bundle validate` subcommand of `operator-sdk` [#3793](https://github.com/operator-framework/operator-sdk/issues/3793). ([#3795](https://github.com/operator-framework/operator-sdk/pull/3795))
```

Comment:
The changelog is automatically generated, so please add a changelog fragment file following this template: https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml

I don't see any docs for that, so I filed https://github.com/operator-framework/operator-sdk/issues/3797